pkgs/development/compilers/dotnet/build-dotnet.nix

```diff
@@ -73,7 +73,7 @@ in stdenv.mkDerivation rec {
     patchelf --set-interpreter "${stdenv.cc.bintools.dynamicLinker}" $out/dotnet
     patchelf --set-rpath "${rpath}" $out/dotnet
     find $out -type f -name "*.so" -exec patchelf --set-rpath '$ORIGIN:${rpath}' {} \;
-    find $out -type f -name "apphost" -exec patchelf --set-interpreter "${stdenv.cc.bintools.dynamicLinker}" --set-rpath '$ORIGIN:${rpath}' {} \;
+    find $out -type f \( -name "apphost" -or -name "createdump" \) -exec patchelf --set-interpreter "${stdenv.cc.bintools.dynamicLinker}" --set-rpath '$ORIGIN:${rpath}' {} \;
```

Comment:
Has someone tried to use `autoPatchelfHook` here? This might avoid future regressions.

Reply:
I tried just now, but a lot of dotnet libraries use `dlopen()`. `runtimeDependencies` doesn't help, because it only adds these to `dotnet` executable itself.

However, while trying that I remembered that dotnet also needs `liblttng-ust` to be able to trace, and an old release at that. Added it and bumped the packages along the way.

Reply:
Ah, so the issue is that `runtimeDependencies` is not applied to shared objects?

Reply:
Correct.